src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/Semantic/SemanticEffectRouter.android.cs

```diff
@@ -24,7 +24,7 @@ protected override void Update(global::Android.Views.View view, SemanticEffectRo
 			var desc = SemanticEffect.GetDescription(Element);
 			var hint = SemanticEffect.GetHint(Element);
 
-			if (!string.IsNullOrEmpty(hint) || !string.IsNullOrEmpty(desc))
+			if (!string.IsNullOrWhiteSpace(hint) || !string.IsNullOrWhiteSpace(desc))
```

Comment:
Can we update this to match the latest changes on .NET MAUI

https://github.com/dotnet/maui/blob/c1f8982721c428d7e03c31a07628aabf371bc699/src/Core/src/Handlers/View/ViewHandler.Android.cs#L117